pyproject.toml

```diff
@@ -4,9 +4,10 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "skfolio"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
no need, version bump is performed in auto by the semantic versioning action (based on commit message)